bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -152,63 +157,70 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
 
         HttpClient client = new HttpClient();
```

Comment:
@kaikreuzer @maggu2810 
We use the HttpClient to store authentication data in the [AuthenticationStore ](http://download.eclipse.org/jetty/stable-9/apidocs/) as well, so I guess I should call ```clearAuthenticationResults()``` on every new request and it should be ok.